style.css

```diff
@@ -0,0 +1,578 @@
+  color: var(--primary-text-color);
+  font-weight: bold;
+  text-transform: capitalize;
+}
+
+li {
+  list-style: none;
+}
+
+button {
+  border: none;
+  outline: none;
+  cursor: pointer;
+  text-transform: capitalize;
+}
+
+svg {
+  width: auto;
+  height: 30px;
+  padding-top: 2%;
+}
+
+h1 {
+  line-height: 1.5em;
+}
+
+nav {
+  padding: 5px;
+  box-shadow: 0 1px 1px 0 rgba(148, 143, 143, 0.5);
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 1000;
+  background-color: #fff;
+}
+
+nav .nav-container {
+  justify-content: space-between;
+}
+
+.nav-icons i {
+  padding: 10px;
+  cursor: pointer;
+  border-radius: 5px;
+  border: 1px solid transparent;
+}
+
+.nav-icons i:hover {
+  background-color: #f7f7f7;
+}
+
+.nav-icons i:active,
+.nav-icons i:focus {
+  border: 1px solid var(--borders);
+}
+
+.strong-link {
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.strong-link:hover {
+  text-decoration: underline;
+}
+
+.nav-btn {
+  background-color: #000;
+  color: var(--primary-btn-text-color);
+  padding: 7px 15px;
+  border-radius: 5px;
+}
+
+.nav-btn:hover {
+  background-color: rgb(51, 51, 51);
+}
+
+.nav-center {
+  align-items: center;
+}
+
+main {
+  position: relative;
+  top: 10vh;
+  width: 65%;
+  margin: 0 auto;
+}
+
+article figure {
+  margin: 0 -20%;
+}
+
+article figure img {
+  height: 500px;
+  margin: 0 auto;
+}
+
+article figure figcaption {
+  width: 75%;
+  padding: 5px 0;
+  font-size: 14px;
+  line-height: 1.5em;
+  color: var(--secondary-text-color);
+}
+
+article figure figcaption span {
+  font-size: 12px;
+
+  /* color: #888; */
+  color: var(--secondary-text-color);
+}
+
+.author-links span {
+  color: var(--primary-text-color);
+}
+
+.author-social {
+  justify-content: space-between;
+}
+
+.author-social a {
+  background-color: #ddd;
+  padding: 5px 6px;
+  border-radius: 50%;
+  border: 1px solid var(--borders);
+}
+
+.article-footer-text,
+.article-footer-text a {
+  color: var(--secondary-text-color);
+  font-size: 12px;
+  line-height: 1.5em;
+  font-weight: normal;
+}
+
+.more-popular a {
+  font-size: 1.5em;
+  font-weight: light;
+  text-decoration: none;
+}
+
+.more-articles-grid-item a {
+  text-decoration: none;
+  font-size: 14px;
+}
+
+.more-articles-grid h3 a {
+  text-decoration: none;
+}
+
+.more-popular li {
+  padding: 20px 0;
+  border-top: 1px solid var(--borders);
+}
+
+footer .footer-list li {
+  padding: 5px 0;
+}
+
+footer .footer-list li a {
+  font-size: 1em;
+  text-decoration: none;
+  font-weight: light;
+  color: var(--secondary-text-color);
+}
+
+.author-social a:hover {
+  background-color: white;
+}
+
+.article-footer-text a:hover {
+  text-decoration: none;
+}
+
+.more-articles-grid-item a:hover {
+  color: var(--secondary-text-color);
+}
+
+.more-popular a:hover {
+  color: var(--secondary-text-color);
+}
+
+.more-articles-grid h3 a:hover {
+  text-decoration: underline;
+}
+
+footer .footer-list li a:hover {
+  text-decoration: underline;
+}
```

Comment:
Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).